akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -241,6 +241,9 @@ final class Flow[-In, +Out, +Mat](private[stream] override val module: Module)
 
   /** Converts this Scala DSL element to it's Java DSL counterpart. */
   def asJava: javadsl.Flow[In, Out, Mat] = new javadsl.Flow(this)
+
+  override def toString =
```

Comment:
I would rather not. String representation could get rather long, so it should probably be better to create these long strings only when needed.